src/components/LicenseCode.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <p xmlns:dct="http://purl.org/dc/terms/" xmlns:cc="http://creativecommons.org/ns#" class="license-text">
+        <i18n path="license-use.richtext.full-text" tag="span">
+            <template v-slot:workTitle>
+                <a :href="workUrl" v-if="workUrl && isWeb" rel="cc:attributionURL">
+                    <span v-if="!workTitle">{{$t('license-use.richtext.workTitle')}}</span>
+                    <span v-else rel="dc:title">{{workTitle}}</span>
+                </a>
+                <span v-else-if="workTitle" rel="dc:title">{{workTitle}}</span>
+                <span v-else>{{ $t('license-use.richtext.workTitle') }}</span>
+            </template>
+            <template v-slot:creator>
+                <a :href="creatorProfileUrl" v-if="creatorProfileUrl && isWeb" rel="cc:attributionURL">
+                    <span v-html="creatorSpan" /></a>
+                <span v-else v-html="creatorSpan" />
+            </template>
+            <template v-slot:by>
+                {{$t(byString)}}
+            </template>
+            <template v-slot:licenseName>
+                <a v-if="isWeb" :href="licenseUrl('web')" target="_blank" rel="license noopener noreferrer"
+                   style="display: inline-block;">{{shortName}}</a>
+                <span v-else>{{shortName}}</span>
+                <LicenseIcons
+                    v-if="isWeb"
+                    :url="licenseUrl"
+                    :iconsArr="iconsList"/>
+            </template>
+        </i18n>
+        <i18n path="license-use.richtext.print-instructions" tag="span" v-if="!isWeb">
+            <template v-slot:linkToLicenseDeed>{{licenseUrl('print')}}</template>
```

Comment:
There is no padding on the text on top or bottom for this option where as richtext has 1rem on top and bottom. I would slightly lessen the padding that rich text has (maybe to .8 or .7rem or something), and add a matching amount of padding to the print LicenseCode text.

No need to add padding to the textarea for the HTML code.

Reply:
Done